cube_builder_aws/cube_builder_aws/forms.py

```diff
@@ -8,10 +8,11 @@
 
 """Define Cube Builder AWS forms used to validate both data input and data serialization."""
 
-from bdc_catalog.models import Collection, GridRefSys, db
+from bdc_catalog.models import Band, Collection, GridRefSys, db
 from marshmallow import Schema, fields, pre_load
 from marshmallow.validate import OneOf, Regexp, ValidationError
-from marshmallow_sqlalchemy.schema import ModelSchema
+from marshmallow_sqlalchemy import auto_field
+from marshmallow_sqlalchemy.schema import ModelSchema, SQLAlchemyAutoSchema
```

Comment:
Just make sure to use a marshmallow-sqlalchemy version which supports ModelSchema. They have removed from 0.23 i think